qlty-check/src/tool/php.rs

```diff
@@ -65,13 +65,15 @@ impl Tool for Php {
     }
 
     fn extra_env_paths(&self) -> Vec<String> {
-        split_paths(
-            &std::env::var("PATH")
-                .with_context(|| "PATH not found for php runtime")
-                .unwrap(),
-        )
-        .map(path_to_native_string)
-        .collect_vec()
+        match std::env::var("PATH") {
+            Ok(path) => split_paths(&path)
+                .map(path_to_native_string)
+                .collect_vec(),
+            Err(_) => {
+                debug!("PATH not found for php runtime, using empty path");
+                Vec::new()
+            }
+        }
```

Comment:
Let's change the return type for `fn extra_env_paths` from `Vec<String>` to `Result<Vec<String>>` in the `Tool` trait. Then, return an `Err` here if the PATH variable is unavailable